core/client/fs/src/main/java/alluxio/client/file/cache/LocalCacheFileInStream.java

```diff
@@ -106,7 +106,7 @@ public int read(ByteBuffer buf, int off, int len) throws IOException {
     byte[] b = new byte[buf.remaining()];
     int totalBytesRead =
         readInternal(b, off, len, ReadType.READ_INTO_BYTE_BUFFER, mPosition, false);
-    buf.put(b, off, len);
+    buf.put(b, off, totalBytesRead);
```

Comment:
If the readInternal return `-1`, how to handle this situation?

Reply:
Thanks for your careful review, I have added one read empty file test case that read -1, PATL.